source/language/types.rst

```diff
@@ -69,6 +69,13 @@ significantly since there is no need for quantum memory management.
 However, it also means that users or compiler have to explicitly manage
 the quantum memory.
 
+.. code-block:: c
+
+   // Declare a qubit
+   qubit gamma;
+   // Declare a qubit with a Unicode name
+   qubit γ;
+
```

Comment:
Pls add the qubit register line: `qubit[20] qubit_array;` Seems you've deleted this

Reply:
added!